security/s2a/internal/crypter/halfconn.go

```diff
@@ -0,0 +1,28 @@
+package crypter
+
+import (
+	"fmt"
+	"golang.org/x/crypto/hkdf"
+	"hash"
+)
+
+// defaultHKDFExpander is the default HKDF expander which uses Go's crypto/hkdf
```

Comment:
s/expander/expansion function,

Reply:
I think keeping expander here might be better than swapping it with expansion function. The class itself is the expander class and its main purpose is to provide an expansion function method. I think if we swap expander for expansion function, we would be implying that the class itself is the expansion function, which might be a bit confusing. WDYT?

Reply:
Sounds reasonable to me. Let's leave it. :)